test/Microsoft.AspNetCore.Mvc.RazorPages.Test/DependencyInjection/RazorPagesOptionsExtensionsTest.cs

```diff
@@ -66,10 +66,39 @@ public void AuthorizePage_AddsAuthorizeFilterToSpecificPage()
                 model => Assert.Empty(model.Filters));
         }
 
+        [Fact]
+        public void AuthorizePage_AddsAuthorizeFilterWithoutPolicyToSpecificPage()
+        {
+            // Arrange
+            var options = new RazorPagesOptions();
+            var models = new[]
+            {
+                new PageApplicationModel("/Pages/Index.cshtml", "/Index.cshtml"),
+                new PageApplicationModel("/Pages/Users/Account.cshtml", "/Users/Account.cshtml"),
+                new PageApplicationModel("/Pages/Users/Contact.cshtml", "/Users/Contact.cshtml"),
+            };
+
+            // Act
+            options.AuthorizePage("/Users/Account.cshtml");
+            ApplyConventions(options, models);
+
+            // Assert
+            Assert.Collection(models,
+                model => Assert.Empty(model.Filters),
+                model =>
+                {
+                    Assert.Equal("/Users/Account.cshtml", model.ViewEnginePath);
+                    var authorizeFilter = Assert.IsType<AuthorizeFilter>(Assert.Single(model.Filters));
+                    var authorizeData = Assert.IsType<AuthorizeAttribute>(Assert.Single(authorizeFilter.AuthorizeData));
+                    Assert.Equal(string.Empty, authorizeData.Policy);
+                },
+                model => Assert.Empty(model.Filters));
+        }
+
         [Theory]
         [InlineData("/Users")]
         [InlineData("/Users/")]
-        public void AuthorizePage_AddsAuthorizeFilterToPagesUnderFolder(string folderName)
+        public void AuthorizePage_AddsAuthorizeFilterWithPolicyToPagesUnderFolder(string folderName)
```

Comment:
We can add optional param for `policy` in this way we can merge both tests in one, let me know if I can go with it